src/components/navbar/Styles.js

```diff
@@ -0,0 +1,71 @@
+import styled from "styled-components";
+
+const red = "#A5000C";
+
+const tablet = "768px";
+const mobile = "480px";
+
+export const TitleContent = styled.div`
+  max-width: 380px;
+  margin: 16px 10px;
+  text-align: left;
+`;
+
+export const HeaderContent = styled.div`
+  max-width: 1440px;
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+export const Title = styled.h1`
+  margin: 0;
+
+  @media screen and (max-width: ${tablet}) {
+    font-size: 1.75rem;
+  }
+
+  @media screen and (max-width: ${mobile}) {
+    font-size: 1rem;
+  }
+`;
+
+export const SubTitle = styled.h2`
+  margin: 0;
```

Comment:
do you need margin 0? seems like the default is 0 according to https://stackoverflow.com/questions/40061984/default-value-of-margin-in-css

Reply:
Yeah, the Chrome browser adds a top and bottom margin.